examples/with-react-hook-form/pages/index.js

```diff
@@ -0,0 +1,5 @@
+import Login from '../components/Login'
+
+const IndexPage = () => <Login />
```

Comment:
Let's just move the component directly in the page, yeah? No need to abstract yet.

Reply:
Yes, you're right. I moved component code into index page